src/test/config/server.config.ts

```diff
@@ -0,0 +1,31 @@
+import dotenv from "dotenv";
+import chaiHttp from "chai-http";
+import chai from "chai";
+import { User } from "../../models/user.model";
+import mongoose from "mongoose";
+import faker from "faker";
+
+dotenv.config();
+process.env.MONGODB_CONNECTION_STRING = "mongodb://localhost:27017/recipe-test";
+
+import app from "../../app";
+
+chai.use(chaiHttp);
+
+export const server = app.listen(app.get("port"), () => {
```

Comment:
I wouldn't override the server for tests. Please use normal server.ts configuration to start the server. The rest like dotenv, env, chai init, app import can be stored for example in `test.config.ts` file, and such file you would import in each .spec.ts file